server/queries/createSurvey.js

```diff
@@ -0,0 +1,24 @@
+const createSurvey = async (
+  surveyObj,
+  newQuestionsArray,
+  surveysCollection,
+) => {
+  /* takes the survey object from the front end, replaces the questions 
+with the ObjectIds from the questions collection, and sends to the DB
+*/
+
+  const newSurveyObj = { ...surveyObj, questions: newQuestionsArray };
+  try {
+    const { acknowledged, surveyId } = await surveysCollection.insertOne(
+      newSurveyObj,
+    );
+  } catch (e) {
+    return new Error(e.message);
+  }
+
+  if (acknowledged === false) {
+    return new Error('Query not acknowledged');
+  } else {
```

Comment:
unnecessary  else if you're just doing a return.

Reply:
I like to always return from functions, but I'll get rid of it

Reply:
No the return is fine, just the else statement is required!
This gives linting errors otherwise